tests/test_config.py

```diff
@@ -247,3 +249,37 @@ def test_log_config_file(mocked_logging_config_module):
     mocked_logging_config_module.fileConfig.assert_called_once_with(
         "log_config", disable_existing_loggers=False
     )
+
+
+@pytest.fixture(params=[0, 1])
+def web_concurrency(request):
+    yield request.param
+    if os.getenv("WEB_CONCURRENCY"):
+        del os.environ["WEB_CONCURRENCY"]
+
+
+@pytest.fixture(params=["127.0.0.1", "127.0.0.2"])
+def forwarded_allow_ips(request):
+    yield request.param
+    if os.getenv("FORWARDED_ALLOW_IPS"):
+        del os.environ["FORWARDED_ALLOW_IPS"]
+
+
+def test_env_file(web_concurrency: int, forwarded_allow_ips: str, caplog):
+    """
+    Test that one can load environment variables using an env file.
+    """
+    with tempfile.NamedTemporaryFile("w+") as fp:
+        content = (
+            f"WEB_CONCURRENCY={web_concurrency}\n"
+            f"FORWARDED_ALLOW_IPS={forwarded_allow_ips}\n"
+        )
+        fp.write(content)
+        fp.seek(0)
+        with caplog.at_level(logging.INFO):
```

Comment:
why not following other tests pattern with config.load ?

Reply:
The test that checks if should reload (above on this file), is not calling it as well. So I thought it was fine?

But I'll add it, sure. 😗

Reply:
Applied.